Services/Administration/OJS.Services.Administration.Business/Contests/ContestsBusinessService.cs

```diff
@@ -370,6 +377,9 @@ private async Task AddIpsToContest(Contest contest, string? mergedIps)
         }
     }
 
+    private void ClearContestCache(Contest contest) =>
+        this.contestParticipantsCacheService.ClearContestCacheByContestId(contest.Id);
```

Comment:
No need for this method.. it's simple enough already.

Reply:
Fixed.